gm4_firework_assemblers/data/firework_assemblers/functions/create.mcfunction

```diff
@@ -0,0 +1,9 @@
+#@s = @e[type=item,nbt=CRAFTING_TABLE] at @s if entity @e[type=item,nbt=FLINT_AND_STEEL,distance=..1] if block ~ ~-1 ~ dropper{CC RECIPE} align xyz positioned ~.5 ~-1 ~.5
+#Run from main
+
+summon armor_stand ~ ~.1 ~ {Small:1,NoGravity:1,Marker:1,Invulnerable:1,Invisible:1,DisabledSlots:2039552,Tags:["gm4_no_edit","gm4_firework_assembler"],Fire:200000,CustomName:'"gm4_firework_assembler"',ArmorItems:[{},{},{},{id:"minecraft:red_concrete_powder",Count:1}]}
+particle minecraft:flash
+playsound minecraft:entity.firework_rocket.blast player @a[distance=..20] ~ ~ ~ 100 1 1
+data merge block ~ ~ ~ {Items:[],CustomName:'"Firework Assembler"'}
+advancement grant @a[distance=..4] only gm4:firework_assemblers
+execute positioned ~ ~1 ~ run kill @e[type=minecraft:item,distance=..1]
```

Comment:
This is killing all items?

Reply:
I noticed that while testing, just forgot to implement a fix - will do